test/e2e/metamask-ui.spec.js

```diff
@@ -701,69 +571,4 @@ describe('MetaMask', function () {
       });
     });
   });
-
-  describe('Approves a custom token from dapp when no gas value is specified', function () {
-    it('approves an already created token', async function () {
-      const windowHandles = await driver.getAllWindowHandles();
-      const extension = windowHandles[0];
-      const dapp = await driver.switchToWindowWithTitle(
-        'E2E Test Dapp',
-        windowHandles,
-      );
-      await driver.closeAllWindowHandlesExcept([extension, dapp]);
-      await driver.delay(regularDelayMs);
-
-      await driver.switchToWindow(dapp);
-      await driver.delay(tinyDelayMs);
-
-      await driver.clickElement({
-        text: 'Approve Tokens Without Gas',
-        tag: 'button',
-      });
-
-      await driver.switchToWindow(extension);
-      await driver.delay(regularDelayMs);
-
-      await driver.wait(async () => {
-        const pendingTxes = await driver.findElements(
-          '.transaction-list__pending-transactions .transaction-list-item',
-        );
-        return pendingTxes.length === 1;
-      }, 10000);
-
-      await driver.waitForSelector({
-        // Selects only the very first transaction list item immediately following the 'Pending' header
-        css: '.transaction-list__pending-transactions .transaction-list__header + .transaction-list-item .list-item__heading',
-        text: 'Approve TST spend limit',
-      });
-
-      await driver.clickElement('.transaction-list-item');
-      await driver.delay(regularDelayMs);
-    });
-
-    it('shows the correct recipient', async function () {
-      await driver.clickElement({
-        text: 'View full transaction details',
-        css: '.confirm-approve-content__small-blue-text',
-      });
-      const permissionInfo = await driver.findElements(
-        '.confirm-approve-content__medium-text',
-      );
-      const recipientDiv = permissionInfo[1];
-      assert.equal(await recipientDiv.getText(), '0x2f318C33...C970');
-    });
-
-    it('submits the transaction', async function () {
-      await driver.delay(veryLargeDelayMs);
-      await driver.clickElement({ text: 'Confirm', tag: 'button' });
-      await driver.delay(regularDelayMs);
-    });
-
-    it('finds the transaction in the transactions list', async function () {
-      await driver.waitForSelector({
-        css: '.transaction-list__completed-transactions .transaction-list-item:first-child .list-item__heading',
-        text: 'Approve TST spend limit',
-      });
-    });
-  });
```

Comment:
Not sure why a lot of these test cases are removed.

Reply:
@jpuri These test cases are removed because it is written in this PR in `test/e2e/tests/custom-token-add-approve.spec.js`. We had a meeting with @PeterYinusa and @seaona about this, and they agreed to remove this test cases from the `metamask-ui.spec.js` and move it to the `custom-token-add-approve.spec.js`.